CHANGES.rst

```diff
@@ -24,6 +24,8 @@ Bug Fixes
 - Function ``median_combine`` now correctly calculates the uncertainty for
   masked ``CCDData``. [#608]
 
+- Function ``combine`` now avoids opening files unnecessarily. [#629, #630]
```

Comment:
The new approach actually doesn't avoid opening the files unnecessarily.

Reply:
I think it still does; without the `copy`, `ccd_list` held a file reference to each file in the list if the files were mem-mapped. The `copy` in the update avoids that, so that at most 1 file is open at a time.

Reply:
I suppose it would be more accurate to say that it doesn't keep files open unnecessarily....

Reply:
> I suppose it would be more accurate to say that it doesn't keep files open unnecessarily....

I like that.